### DISCUSSION TITLE
Navigation bars logo interaction in web

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#790] [Proposals]
Actualmente el componente `MainNavigationBar` y `FunnelNavigationBar` no tienen la posibilidad de añadirle al logotipo una url, por ejemplo para marcarle que vaya a la home.

En el ejemplo de vivo https://hub.staging.vivo.segdev.com.br/ usan un "Home" para marcar la home y el logo no tiene ningún tipo de interacción.

No debería tener este componente la capacidad de pasarle una url al logo?

**Comment by AnaMontes11** [comment]
Totalmente. El logo debería llevar a la "home".
